transformer_engine/pytorch/fp8.py

```diff
@@ -496,16 +520,18 @@ def get_fp8_te_dtype(
 
 
 def reduce_tensor_across_group_op_max(
-    tensor: torch.Tensor, group: dist_group_type
+    tensor: torch.Tensor, group: dist_group_type, async_amax_reduction: bool
```

Comment:
```suggestion
    tensor: torch.Tensor, group: dist_group_type, async_op: bool
```

Reply:
This is a general function for reducing a tensor across the given group so the argument names should not be specific to amax reduction.

Reply:
Updated